Explorer/Assets/DCL/UserInAppInitializationFlow/IUserInAppInitializationFlow.cs

```diff
@@ -10,6 +10,7 @@ UniTask ExecuteAsync(
             bool showAuthentication,
             bool showLoading,
             bool reloadRealm,
```

Comment:
This function is getting unweildly I would suggest we introduce a parameter object to improve the readability and if its changed in future it will not require changes to many function calls

Reply:
Good point! I have just applied your suggested change [here](https://github.com/decentraland/unity-explorer/pull/2295/commits/c18f3d7e8401d5169f0b8236bd7154a9946a68b1). Let me know wdyt!